### ISSUE TITLE
Remove types?

### ISSUE THREAD

**Issue by andig** [evcc-io/docs#118]
@naltatis was machen wir mit diesen Listen: https://docs.evcc.io/docs/reference/configuration/chargers

Die sind nicht aktuell und ohne Parameter eigentlich auch nicht hilfreich. Entfernen???

**Comment by andig** [comment]
Siehe auch https://github.com/evcc-io/evcc/issues/3455, das führt zu Verwirrung.

**Comment by naltatis** [comment]
Ja, ich würde vorschlagen, dass wir als öffentliche Schnittstelle nur noch `custom` und `template` hier in der Doku haben. Die direkte Nutzung der andern Typen würde ich inzwischen als umdokumentierte Private API sehen.

**Comment by VolkerK62** [comment]
Kurze Nachfrage: hier  https://docs.evcc.io/docs/devices/chargers  sind alle aufgeführt, die implementiert sind und für alle gibt es auch ein entsprechendes template?  Oder sollte ich das vorher kontrollieren?  Der entsprechende Link ist ja hier https://docs.evcc.io/docs/reference/configuration/chargers  schon enthalten. ich würde dann die Liste kürzen bis auf `custom`.  Bei einigen gibt es Hinweise z.B.  "CFos braucht ein meter", derartige Hinweise würde ich dann in die Geräteliste übernehmen. Oder ist das nicht nötig, da das Template entsprechend gebaut ist?

**Comment by VolkerK62** [comment]
und was mir gerade noch aufgefallen ist, eigentlich sollte das gleiche Prinzip mit dem Templates doch auch bei  https://docs.evcc.io/docs/reference/configuration/meters   und https://docs.evcc.io/docs/reference/configuration/vehicles  angewendet werden, oder?

**Comment by andig** [comment]
> Kurze Nachfrage: hier https://docs.evcc.io/docs/devices/chargers sind alle aufgeführt, die implementiert sind und für alle gibt es auch ein entsprechendes template? Oder sollte ich das vorher kontrollieren? Der entsprechende Link ist ja hier https://docs.evcc.io/docs/reference/configuration/chargers schon enthalten. ich würde dann die Liste kürzen bis auf `custom`. Bei einigen gibt es Hinweise z.B. "CFos braucht ein meter", derartige Hinweise würde ich dann in die Geräteliste übernehmen. Oder ist das nicht nötig, da das Template entsprechend gebaut ist?

Die Templates dienen eigentlich "nur" dazu, die internen Typen mit Zusatzinformationen für die Konfiguration anzureichern, als auch komplexere Plugins "zusammenzustöpseln". Bisher ist noch nicht jedes Gerät in ein Template verpackt. Bei manchen (`movingaverage` meter) ist das auch nicht sinnvoll möglich.

Alle die schon mit einem Template hinterlegt sind können aber von der types Seite verschwinden!

**Comment by VolkerK62** [comment]
Ich habe keinen Type gefunden, der nicht in der Geräteliste aufgeführt ist. Ich habe den Hinweis auf die Geräteliste erweitert und nur noch `custom` drin gelassen.
